.github/workflows/pypi-release.yml

```diff
@@ -143,10 +144,11 @@ jobs:
           path: .
 
       - name: "Publish ${{ needs.sanitize-package-name.outputs.name }} v${{ inputs.version_number }} To Test PyPI"
-        uses: pypa/gh-action-pypi-publish@v1.8.5
+        uses: pypa/gh-action-pypi-publish@v1.12.4
         with:
           password: ${{ secrets.TEST_PYPI_API_TOKEN }}
-          repository_url: https://test.pypi.org/legacy/
+          repository-url: https://test.pypi.org/legacy/
```

Comment:
You could make this an environment variable in `PypiTest` and you might be able to collapse these two. You would need to add the default version in `PypiProd`, but that's a small price to reduce redundancy.

Reply:
Do the adapters pre-mono-repo work still use this process?  Or are they all on your own release?  The reason I never did this is because that has to be set up in each repo (core + adapters) and is actually more duplicate to do that way.

Reply:
Oh, it does. But at this point, we only would use it for 1.8 and earlier. But since the environment is called in this workflow, would it matter? Wouldn't it use the environment from this repo? Or does it use the environment from the calling repo?

Reply:
It used the env from the calling repo.  workflow_dispatch runs the workflow in the context of the calling workflow.  That's why I never did this - it was a lot of repetition to set up everywhere.  I'd prefer to leave for now in the case we need to do any of those releases for any reason.